AmplifyPlugins/Storage/AWSS3StoragePluginTests/Operation/AWSS3StorageGetDataOperationTests.swift

```diff
@@ -38,7 +38,7 @@ class AWSS3StorageDownloadDataOperationTests: AWSS3StorageOperationTestBase {
     }
 
     func testDownloadDataOperationGetIdentityIdError() {
-        mockAuthService.getIdentityIdError = AuthError.identity("", "", "")
+        mockAuthService.getIdentityIdError = AuthError.service("", "", "")
```

Comment:
this change will probably break the storage integration tests since there are access level tests which tests assert for an "identity" error, referencing here https://github.com/aws-amplify/amplify-ios/issues/435

Reply:
Thanks for adding the issue, will handle this in another PR.